data_tools/writeSample.py

```diff
@@ -28,6 +28,17 @@ def people(fbConfig):
     print(len(people), "people updated successfully")
 
 
+def messages(fbConfig):
+    messages = generateSample.messages(10)
+    messagesCollection = fb.db.collection(fbConfig["chartCollection"]).document(
+        fbConfig["dataDoc"]).collection(fbConfig["messagesCollection"])
+    i = 0
+    for message in messages:
```

Comment:
In general dual iterators like this are a bit frowned on, unless you really have to. Consider:

```
for i in range(0, len(messages)):
  messagesCollection.document(str(i)).set({"messages": messages[i]})
```

or the like

Reply:
Ah got it! Fixed